teku/src/main/java/tech/pegasys/teku/cli/options/DataOptions.java

```diff
@@ -34,6 +34,25 @@
       arity = "1")
   private StateStorageMode dataStorageMode = StateStorageMode.PRUNE;
 
+  @Option(
+      names = {"--data-storage-archive-frequency"},
+      hidden = true,
+      paramLabel = "<FREQUENCY>",
+      description =
+          "Sets the frequency at which to store archived slots to disk. (slot modulo frequency == 0)",
```

Comment:
We're storing archived states, not slots.
```suggestion
          "Sets the frequency, in slots, at which to store archived states to disk.",
```